rastervision_pytorch_learner/rastervision/pytorch_learner/learner_config.py

```diff
@@ -467,6 +467,15 @@ class GeoDataWindowConfig(Config):
         description='Max attempts when trying to find a window within the AOI '
         'of a scene. Only used if method = random and the scene has '
         'aoi_polygons specified.')
+    efficient_aoi_samplings: bool = Field(
```

Comment:
This is mostly here to provide a way to turn this functionality off on the off chance we encounter an input that the triangulation algorithm cannot handle.

Reply:
Oops. The 's' at the end is a typo. This supposed to be called `efficient_aoi_sampling` not `efficient_aoi_samplings`.